CSNotificationView/CSNotificationView.m

```diff
@@ -383,6 +383,10 @@ - (void)dismissWithStyle:(CSNotificationViewStyle)style message:(NSString *)mess
 - (CGFloat)topLayoutGuideLengthCalculation
 {
     CGFloat top = MIN([UIApplication sharedApplication].statusBarFrame.size.height, [UIApplication sharedApplication].statusBarFrame.size.width);
+
+    UIView *view = self.parentNavigationController.view ?: self.parentViewController.view;
+    CGPoint origin = [view.superview convertPoint:view.frame.origin toView:[UIApplication sharedApplication].keyWindow.rootViewController.view];
```

Comment:
Actually in my case it's not a modal popover on iPad but a [custom content view controller] (https://developer.apple.com/library/ios/featuredarticles/ViewControllerPGforiPhoneOS/CreatingCustomContainerViewControllers/CreatingCustomContainerViewControllers.html).

My logic was that the status bar height should only be added to the ´topLayoutGuide´ when the view controller frame actually has an intersection with the status bar frame. I haven't got an idea how to achieve that without using ´convertRect:fromView:´ or something similar.